src/puppetlabs/puppetdb/utils.clj

```diff
@@ -257,3 +257,10 @@
       {:href (to-href data)}
       (-> (sutils/parse-db-json data)
           (update :href to-href)))))
+
+(defn hsql?
+  "given the globals configuration bundle, determine if hsqldb is being used."
+  [globals]
```

Comment:
I mean we'd just call it `db` but all the other functions for this kind of thing I've seen outside this project use the `db` (`classname`, `subprotocol`, `subname`) as the param